data-prepper-plugin-framework/src/main/java/org/opensearch/dataprepper/plugin/ObjectMapperConfiguration.java

```diff
@@ -33,6 +34,7 @@ public class ObjectMapperConfiguration {
     ObjectMapper extensionPluginConfigObjectMapper() {
         final SimpleModule simpleModule = new SimpleModule();
         simpleModule.addDeserializer(Duration.class, new DataPrepperDurationDeserializer());
+        simpleModule.addDeserializer(Enum.class, new EnumDeserializer());
```

Comment:
This will be applied to every Enum defined in the project?

Reply:
Yes it will be

Reply:
Any configuration file that has a variable that is an Enum

Reply:
nice 👍